docs/sparkr.md

```diff
@@ -14,29 +14,24 @@ supports operations like selection, filtering, aggregation etc. (similar to R da
 [dplyr](https://github.com/hadley/dplyr)) but on large datasets. SparkR also supports distributed
 machine learning using MLlib.
 
-# SparkR DataFrames
+# SparkDataFrames
```

Comment:
Oh, @felixcheung 
This will make broken links.
For example, `DataFrame.R` line 38,
```
@seealso \url{https://spark.apache.org/docs/latest/sparkr.html#sparkr-dataframes}
```
You should fix together. I'm not sure the other links outside of the R module.